src/index.ts

```diff
@@ -209,6 +222,60 @@ export function buildStorageKeySuffix(apiKey: string): string {
   return apiKey.replace(/\W/g, '').substring(0, 8);
 }
 
+/**
+ * Initializes the Eppo client with configuration parameters.
+ *
+ * The purpose is for use-cases where the configuration is available from an external process
+ * that can bootstrap the SDK.
+ *
+ * This method should be called once on application startup.
+ *
+ * @param config - client configuration
+ * @returns a singleton client instance
+ * @public
+ */
+export function initSync(config: IClientConfigSync): IEppoClient {
```

Comment:
`syncInit` instead?

Reply:
Honestly I don't like either - seems a little on the nose: should be obvious its sync from the `lack` of `async` in the function definition. Let me ponder something more elegant...

Reply:
yeah I think we may need a completely different name here to capture the fact that this initializes "offline" and doenst make any network requests. Maybe `initFromPrefetchedConfig`? Something more explicit would be better IMHO

Reply:
`offlineInit`?

Reply:
That's a good one, thanks!